Dockerfile

```diff
@@ -10,8 +10,6 @@ COPY . /usr/src
 # install dependencies
 RUN npm install
 
-
 # start app
 RUN npm run build
-EXPOSE 3000
```

Comment:
If you don't expose a port, many cloud app platforms won't know what to do by default. What does it hurt to leave the expose clause here?